app/src/main/java/io/left/hellomesh/MainActivity.java

```diff
@@ -47,7 +47,10 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        mm = AndroidMeshManager.getInstance(MainActivity.this, MainActivity.this);
+        // TODO: set pattern
```

Comment:
Could this be more descriptive as an instruction so it doesn't look like we forgot something? Like "set a wifi pattern by passing it as the third arugment" or something like that?

Reply:
Changed